### PR TITLE
Add german translation v2.0

### PR DESCRIPTION
**Pull request by mheob**
This is the translation of the Code of Conduct into German.
I have tried to use version 1.4 as a reference to keep the differences as small as possible.

### CODE REVIEW

**Comment by CoralineAda** [comment]
Thank you so much! I'll put out a call for other native speakers to review.

File: content/version/2/0/code_of_conduct.de.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmer und Verantwortliche unserer Gemeinschaft
+verpflichten uns bei der Teilnahme an dem Projekt und unserer Gemeinschaft zu
+einer belästigungsfreien Erfahrung für jeden, unabhängig von Alter, Körpergröße,
+Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck,
+Erfahrungsstufe, Bildung, sozialem Status, Nationalität, persönlicher
+Erscheinung, Rasse, Religion oder sexuelle Identität und Orientierung.
+
```
Comment (danielaKay) [inline content/version/2/0/code_of_conduct.de.md:17]: sexuelle_r_ Identität

File: content/version/2/0/code_of_conduct.de.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+einer offenen, einladenden, vielfältigen, integrativen und gesunden Gemeinschaft
+beiträgt.
+
+## Unsere Standards
+
+Beispiele für Verhaltensweisen, die zu einem positiven Umfeld für unsere
+Gemeinschaft beitragen, sind:
+
+- Einfühlungsvermögen und Freundlichkeit gegenüber anderen Menschen zeigen
+- Respekt vor unterschiedlichen Meinungen, Standpunkten und Erfahrungen
```
Comment (danielaKay) [inline content/version/2/0/code_of_conduct.de.md]: all other points in this section end with a verb. Is this possible here for consistency's sake?
(very minor point, feel free to ignore)

**Comment by raketenlurch** [comment]
I'm an native speaker and could have a look at this this evening. :)

File: content/version/2/0/code_of_conduct.de.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+Die Gemeinschaftsleiter sind verantwortlich dafür, unsere Standards für ein
+akzeptables Benehmen klarzustellen und werden angemessen und fair
+korrigierende Maßnahmen ergreifen um auf jegliches Verhalten, das sie für
+unangemessen, bedrohlich oder beleidigend halten, zu reagieren.
+
+Die Gemeinschaftsleiter haben das Recht und die Verantwortung Kommentare,
+Commits, Code, Wiki-Bearbeitungen, Support-Tickets und andere Beiträge, die
+nicht mit diesem Verhaltenskodex vereinbar sind zu entfernen, zu bearbeiten oder
+abzulehnen, und wird die Gründe für eine Mäßigung mitteilen Entscheidungen, wenn
```
Comment (danielaKay) [inline content/version/2/0/code_of_conduct.de.md]: is the text after "mitteilen" correct? There seem to be words missing...?

File: content/version/2/0/code_of_conduct.de.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+Beispiele für die Repräsentation unserer Gemeinschaft sind die Verwendung einer
+offiziellen E-Mail-Adresse, das Posten über ein offizielles Social-Media-Konto
+oder das Auftreten als ernannter Vertreter bei einer Online- oder
+Offline-Veranstaltung.
+
+## Umsetzung
+
+Fälle von missbräuchlichem, belästigendem oder anderweitig inakzeptablem
+Verhalten können unter [KONTAKTMETHODE EINFÜGEN] den für die Durchsetzung
+zuständigen Gemeinschaftsleiter gemeldet werden. Alle Beschwerden werden zeitnah
```
Comment (danielaKay) [inline content/version/2/0/code_of_conduct.de.md]: either "dem ... Gemeinschaftsleiter" oder "den ... Gemeinschaftsleitern"

File: content/version/2/0/code_of_conduct.de.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+und fair geprüft und ermittelt.
+
+Alle Gemeinschaftsleiter sind verpflichtet, die Privatsphäre und Sicherheit des
+Berichterstatters eines Vorfalls zu respektieren.
```
Comment (danielaKay) [inline content/version/2/0/code_of_conduct.de.md]: "Berichterstatter" feels a bit awkward here. Maybe it's better to use something like "desjenigen, der einen Vorfall gemeldet hat"?

**Review by danielaKay — COMMENTED** [review]
Great job with the translation! 
I added 5 comments with suggestions for minor changes.

File: content/version/2/0/code_of_conduct.de.md (hunk 1 above)
Comment (DennisTraub) [reply content/version/2/0/code_of_conduct.de.md:17]: I agree with @danielaKay. However, I would even suggest replacing "... oder sexueller Identität und Orientierung" with "... oder Geschlechtsidentität und sexueller Orientierung". The term _Geschlechtsidentität_ expresses that there's more to gender identity than just the sexual aspect.

**Comment by mheob** [comment]
Many thanks to @danielaKay and @DennisTraub for the suggested changes! I have added them all.

File: content/version/2/0/code_of_conduct.de.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+Die Gemeinschaftsleiter sind verantwortlich dafür, unsere Standards für ein
+akzeptables Benehmen klarzustellen und werden angemessen und fair
+korrigierende Maßnahmen ergreifen um auf jegliches Verhalten, das sie für
+unangemessen, bedrohlich oder beleidigend halten, zu reagieren.
+
+Die Gemeinschaftsleiter haben das Recht und die Verantwortung Kommentare,
+Commits, Code, Wiki-Bearbeitungen, Support-Tickets und andere Beiträge, die
+nicht mit diesem Verhaltenskodex vereinbar sind zu entfernen, zu bearbeiten oder
+abzulehnen, und wird die Gründe für eine Mäßigung mitteilen, wenn es angebracht
```
Comment (sduslu) [inline content/version/2/0/code_of_conduct.de.md]: "Die Gemeinschaftsleiter ... werden die Gründe ... mitteilen ..."
(werden instead of wird, because Gemeinschaftsleiter are plural)

File: content/version/2/0/code_of_conduct.de.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+## Umsetzung
+
+Fälle von missbräuchlichem, belästigendem oder anderweitig inakzeptablem
+Verhalten können unter [KONTAKTMETHODE EINFÜGEN] den für die Durchsetzung
+zuständigen Gemeinschaftsleitern gemeldet werden. Alle Beschwerden werden
+zeitnah und fair geprüft und ermittelt.
+
+Alle Gemeinschaftsleiter sind verpflichtet, die Privatsphäre und Sicherheit
+desjenigen, der einen Vorfall gemeldet hat, zu respektieren.
```
Comment (sduslu) [inline content/version/2/0/code_of_conduct.de.md]: Maybe rephrase for not implying male gender (desjeningen, der) by:
"... die Sicherheit derjenigen, die einen Vorfall gemeldet haben, zu respektieren"

**Review by sduslu — COMMENTED** [review]
Nice translation, just a little comment from my side (native German speaker). Thanks for the good job!

File: content/version/2/0/code_of_conduct.de.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/de"]
++++
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmer und Verantwortliche unserer Gemeinschaft
```
Comment (raketenlurch) [inline content/version/2/0/code_of_conduct.de.md]: I would suggest to use "Teilnehmende" instead of "Teilnehmer" because it's more gender neutral.

File: content/version/2/0/code_of_conduct.de.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+  Nutzung des Internets für unbilliges Verhalten
+- Öffentliche oder private Belästigungen
+- Das Veröffentlichen von privaten Informationen Anderer, wie zum Beispiel
+  physische oder elektronische Adressen, ohne deren ausdrückliche Erlaubnis
+- Anderes Verhalten, welches in einem professionellen Umfeld begründet als
+  unangemessen betrachtet werden kann
+
+## Verantwortlichkeiten bei der Durchsetzung
+
+Die Gemeinschaftsleiter sind verantwortlich dafür, unsere Standards für ein
```
Comment (raketenlurch) [inline content/version/2/0/code_of_conduct.de.md]: Here I would suggest "Gemeinschaftsleitende" instead of "Gemeinschaftsleiter". Also because it's more gender-neutral.

File: content/version/2/0/code_of_conduct.de.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+akzeptables Benehmen klarzustellen und werden angemessen und fair
+korrigierende Maßnahmen ergreifen um auf jegliches Verhalten, das sie für
+unangemessen, bedrohlich oder beleidigend halten, zu reagieren.
+
+Die Gemeinschaftsleiter haben das Recht und die Verantwortung Kommentare,
```
Comment (raketenlurch) [inline content/version/2/0/code_of_conduct.de.md]: Same as above.

File: content/version/2/0/code_of_conduct.de.md [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+Offline-Veranstaltung.
+
+## Umsetzung
+
+Fälle von missbräuchlichem, belästigendem oder anderweitig inakzeptablem
+Verhalten können unter [KONTAKTMETHODE EINFÜGEN] den für die Durchsetzung
+zuständigen Gemeinschaftsleitern gemeldet werden. Alle Beschwerden werden
+zeitnah und fair geprüft und ermittelt.
+
+Alle Gemeinschaftsleiter sind verpflichtet, die Privatsphäre und Sicherheit
```
Comment (raketenlurch) [inline content/version/2/0/code_of_conduct.de.md]: Same as above.

File: content/version/2/0/code_of_conduct.de.md [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+desjenigen, der einen Vorfall gemeldet hat, zu respektieren.
+
+## Durchsetzungsrichtlinien
+
+Die Gemeinschaftsleiter werden sich bei der Bestimmung der Konsequenzen für jede
```
Comment (raketenlurch) [inline content/version/2/0/code_of_conduct.de.md]: Same as above.

File: content/version/2/0/code_of_conduct.de.md [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+Handlung, die ihrer Ansicht nach gegen diesen Verhaltenskodex verstößt, an diese
+Richtlinien über die Auswirkungen in der Gemeinschaft halten:
+
+### 1. Berichtigung
+
+**Auswirkungen in der Gemeinschaft**: Verwendung unangemessener Sprache oder
+anderes Verhalten, das in der Gemeinschaft als unprofessionell oder unwillkommen
+gilt.
+
+**Folge**: Eine private, schriftliche Verwarnung von den Gemeinschaftsleitern,
```
Comment (raketenlurch) [inline content/version/2/0/code_of_conduct.de.md]: Same as above.

**Comment by raketenlurch** [comment]
I left a few comments. Otherwise this looks good to me. :)

**Comment by mheob** [comment]
Many thanks also to @sduslu and @raketenlurch for the proposed changes! I have also added them all.
I especially like the gender-neutral renaming.

File: content/version/2/0/code_of_conduct.de.md [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+aliases = ["/version/2/0/de"]
++++
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmende und Verantwortliche unserer Gemeinschaft
+verpflichten uns bei der Teilnahme an dem Projekt und unserer Gemeinschaft zu
+einer belästigungsfreien Erfahrung für jeden, unabhängig von Alter, Körpergröße,
```
Comment (ImmoStanke) [inline content/version/2/0/code_of_conduct.de.md]:
```suggestion
einer belästigungsfreien Erfahrung für alle, unabhängig von Alter, Körpergröße,
```
Another suggestion for more gender neutral language.

**Comment by mheob** [comment]
Thanks @ImmoStanke! I also add your suggestion.

**Comment by ImmoStanke** [comment]
Looks good (enough) to me and could be merged in my opinion. 

The only thing that's a bit off semantically is that in the English version of the first paragraph, there's a distinction between:
* sex characteristics
* gender identity
* gender expression
and 
* sexual identity

In the German translation this turns into:
* geschlechtliche Identität
* geschlechtlicher Ausdruck
and
* Geschlechtsidentität

Because there is no easy verbal distinction between 'sex' and 'gender' in German it both translates to 'Geschlecht'.
I think there's two ways to approach this:
Either drop "Geschlechtsidentität" since it's basically listed twice now, or find some way to convey the distinction that the English version draws (but I'm not sure how without loosing conciseness). I guess it depends if the goal is more to stay as close as possible to the Original, or to find the most natural translation. The most natural translation would mean loosing this distinction, which is unfortunately hard to draw in German. The most accurate translation would need to find some way to draw the distinction anyways even at the cost of being harder to understand, longer, or awkwardly worded.
As mentioned above, I don't think this should stop this from being merged but maybe it's a conversation worth having (possibly in a separate issue).

File: content/version/2/0/code_of_conduct.de.md [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmende und Verantwortliche unserer Gemeinschaft
+verpflichten uns bei der Teilnahme an dem Projekt und unserer Gemeinschaft zu
+einer belästigungsfreien Erfahrung für alle, unabhängig von Alter, Körpergröße,
+Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck,
+Erfahrungsstufe, Bildung, sozialem Status, Nationalität, persönlicher
```
Comment (hubx) [inline content/version/2/0/code_of_conduct.de.md]: minor: `Erfahrungsstufe` sounds a bit out of context here. rather `Grad der Erfahrung` or `Erfahrungsgrad`

File: content/version/2/0/code_of_conduct.de.md [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+Erscheinung, Rasse, Religion oder Geschlechtsidentität und sexueller
+Orientierung.
+
+Wir verpflichten uns, in einer Weise zu handeln und zu interagieren, die zu
+einer offenen, einladenden, vielfältigen, integrativen und gesunden Gemeinschaft
```
Comment (hubx) [inline content/version/2/0/code_of_conduct.de.md]: original says `inclusive`, so I rather `inklusiv` instead of `integrativen` to keep the original association of `Inklusion`

File: content/version/2/0/code_of_conduct.de.md [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+  Fehlern betroffen sind, und aus den Erfahrungen lernen
+- Konzentration auf das, was nicht nur für uns als Individuen, sondern für die
+  gesamte Gemeinschaft das Beste ist
+
+Beispiele für inakzeptables Verhalten sind:
+
+- Die Verwendung sexualisierter Sprache, Bilder oder Symbolik sowie unerwünschte
+  Versuche sexueller Anbahnung
+- Beleidigende oder abwertende Kommentare, persönliche oder politische Angriffe,
+  Nutzung des Internets für unbilliges Verhalten
```
Comment (hubx) [inline content/version/2/0/code_of_conduct.de.md]: What is `Nutzung des Internets für unbilliges Verhalten` referring to trolling? I think there is no translation for trolling

File: content/version/2/0/code_of_conduct.de.md [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+abzulehnen, und werden die Gründe für eine Mäßigung mitteilen, wenn es
+angebracht ist.
+
+## Geltungsbereich
+
+Dieser Verhaltenskodex gilt für alle Gemeinschaftsbereiche und gilt auch, wenn
+eine Einzelperson die Gemeinschaft offiziell in öffentlichen Bereichen vertritt.
+Beispiele für die Repräsentation unserer Gemeinschaft sind die Verwendung einer
+offiziellen E-Mail-Adresse, das Posten über ein offizielles Social-Media-Konto
+oder das Auftreten als ernannter Vertreter bei einer Online- oder
```
Comment (hubx) [inline content/version/2/0/code_of_conduct.de.md]: gender neutral: `Vertreter` to `als ernannte Vertretung` also more aligned with `Repräsentation` instead of `Repräsentant` above

File: content/version/2/0/code_of_conduct.de.md [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+Verhalten können unter [KONTAKTMETHODE EINFÜGEN] den für die Durchsetzung
+zuständigen Gemeinschaftsleitenden gemeldet werden. Alle Beschwerden werden
+zeitnah und fair geprüft und ermittelt.
+
+Alle Gemeinschaftsleitenden sind verpflichtet, die Privatsphäre und die
+Sicherheit derjenigen, die einen Vorfall gemeldet haben, zu respektieren.
+
+## Durchsetzungsrichtlinien
+
+Die Gemeinschaftsleitenden werden sich bei der Bestimmung der Konsequenzen für
```
Comment (hubx) [inline content/version/2/0/code_of_conduct.de.md]: I saw this already changed recently. Just want to mention another option instead of descriptive language is `Gemeinschaftsleitung` - more a style question here.

**Review by hubx — COMMENTED** [review]
another review

File: content/version/2/0/code_of_conduct.de.md (hunk 19 above)
Comment (mheob) [reply content/version/2/0/code_of_conduct.de.md]: I also like `Gemeinschaftsleitung` more, now that I see it.
But here I would like to see some opinions again. Please 👍🏽  / 👎🏽

File: content/version/2/0/code_of_conduct.de.md (hunk 17 above)
Comment (mheob) [reply content/version/2/0/code_of_conduct.de.md]: This point has not changed from 1.4 to 2.0 (except that the `/` becomes an `or`). So I left it as it was in the previous version.

I'm open to suggestions on how the word `trolling` can be integrated when needed.

**Comment by mheob** [comment]
Thanks @hubx. Except the two points, where I would like to have one or two more opinions, I have added our suggestions.

File: content/version/2/0/code_of_conduct.de.md [hunk 20]
```diff
@@ -0,0 +1,137 @@
...
+- Konstruktives Feedback geben und würdevoll annehmen
+- Verantwortung übernehmen und uns bei denjenigen entschuldigen, die von unseren
+  Fehlern betroffen sind, und aus den Erfahrungen lernen
+- Konzentration auf das, was nicht nur für uns als Individuen, sondern für die
+  gesamte Gemeinschaft das Beste ist
+
+Beispiele für inakzeptables Verhalten sind:
+
+- Die Verwendung sexualisierter Sprache, Bilder oder Symbolik sowie unerwünschte
+  Versuche sexueller Anbahnung
```
Comment (rohte) [inline content/version/2/0/code_of_conduct.de.md]: Anbahnung sounds odd to me here. Would choose Annäherung.

Comment (mheob) [reply content/version/2/0/code_of_conduct.de.md]: I have kept the spelling from 1.4 here, even though it sounds odd to me.
Here I would also like to ask for a short vote: 👍🏽 / 👎🏽

File: content/version/2/0/code_of_conduct.de.md (hunk 17 above)
Comment (hubx) [reply content/version/2/0/code_of_conduct.de.md]: how about "und trollen"
https://de.wikipedia.org/wiki/Troll_(Netzkultur)

File: content/version/2/0/code_of_conduct.de.md (hunk 20 above)
Comment (mheob) [reply content/version/2/0/code_of_conduct.de.md]: After the quick vote I added your suggestion. Thank you for it.

File: content/version/2/0/code_of_conduct.de.md (hunk 19 above)
Comment (mheob) [reply content/version/2/0/code_of_conduct.de.md]: After the quick vote I added your suggestion. Thank you for it.

File: content/version/2/0/code_of_conduct.de.md (hunk 17 above)
Comment (mheob) [reply content/version/2/0/code_of_conduct.de.md]: I added it. Thank you.

File: content/version/2/0/code_of_conduct.de.md [hunk 21]
```diff
@@ -0,0 +1,137 @@
...
+  und trollen
+- Öffentliche oder private Belästigungen
+- Das Veröffentlichen von privaten Informationen Anderer, wie zum Beispiel
+  physische oder elektronische Adressen, ohne deren ausdrückliche Erlaubnis
+- Anderes Verhalten, welches in einem professionellen Umfeld begründet als
+  unangemessen betrachtet werden kann
+
+## Verantwortlichkeiten bei der Durchsetzung
+
+Die Gemeinschaftsleitung sind verantwortlich dafür, unsere Standards für ein
```
Comment (ImmoStanke) [inline content/version/2/0/code_of_conduct.de.md]:
```suggestion
Die Gemeinschaftsleitung ist verantwortlich dafür, unsere Standards für ein
```
If you use "Gemeinschaftsleitung", it's singular, so you need to adjust the case here

File: content/version/2/0/code_of_conduct.de.md [hunk 22]
```diff
@@ -0,0 +1,137 @@
...
+akzeptables Benehmen klarzustellen und werden angemessen und fair
+korrigierende Maßnahmen ergreifen um auf jegliches Verhalten, das sie für
+unangemessen, bedrohlich oder beleidigend halten, zu reagieren.
+
+Die Gemeinschaftsleitung haben das Recht und die Verantwortung Kommentare,
```
Comment (ImmoStanke) [inline content/version/2/0/code_of_conduct.de.md]:
```suggestion
Die Gemeinschaftsleitung hat das Recht und die Verantwortung Kommentare,
```
See above

File: content/version/2/0/code_of_conduct.de.md [hunk 23]
```diff
@@ -0,0 +1,137 @@
...
+Offline-Veranstaltung.
+
+## Umsetzung
+
+Fälle von missbräuchlichem, belästigendem oder anderweitig inakzeptablem
+Verhalten können unter [KONTAKTMETHODE EINFÜGEN] den für die Durchsetzung
+zuständigen Gemeinschaftsleitung gemeldet werden. Alle Beschwerden werden
+zeitnah und fair geprüft und ermittelt.
+
+Alle Gemeinschaftsleitung sind verpflichtet, die Privatsphäre und die
```
Comment (ImmoStanke) [inline content/version/2/0/code_of_conduct.de.md]:
```suggestion
Alle Gemeinschaftsleitenden sind verpflichtet, die Privatsphäre und die
```
or
```suggestion
Die Gemeinschaftsleitung ist verpflichtet, die Privatsphäre und die
```
or even
```suggestion
Die gesamte Gemeinschaftsleitung ist verpflichtet, die Privatsphäre und die
```

File: content/version/2/0/code_of_conduct.de.md [hunk 24]
```diff
@@ -0,0 +1,137 @@
...
+eine Einzelperson die Gemeinschaft offiziell in öffentlichen Bereichen vertritt.
+Beispiele für die Repräsentation unserer Gemeinschaft sind die Verwendung einer
+offiziellen E-Mail-Adresse, das Posten über ein offizielles Social-Media-Konto
+oder das Auftreten als ernannte Vertretung bei einer Online- oder
+Offline-Veranstaltung.
+
+## Umsetzung
+
+Fälle von missbräuchlichem, belästigendem oder anderweitig inakzeptablem
+Verhalten können unter [KONTAKTMETHODE EINFÜGEN] den für die Durchsetzung
```
Comment (ImmoStanke) [inline content/version/2/0/code_of_conduct.de.md]:
```suggestion
Verhalten können unter [KONTAKTMETHODE EINFÜGEN] der für die Durchsetzung
```

File: content/version/2/0/code_of_conduct.de.md [hunk 25]
```diff
@@ -0,0 +1,137 @@
...
+zuständigen Gemeinschaftsleitung gemeldet werden. Alle Beschwerden werden
+zeitnah und fair geprüft und ermittelt.
+
+Alle Gemeinschaftsleitung sind verpflichtet, die Privatsphäre und die
+Sicherheit derjenigen, die einen Vorfall gemeldet haben, zu respektieren.
+
+## Durchsetzungsrichtlinien
+
+Die Gemeinschaftsleitung werden sich bei der Bestimmung der Konsequenzen für
```
Comment (ImmoStanke) [inline content/version/2/0/code_of_conduct.de.md]:
```suggestion
Die Gemeinschaftsleitung wird sich bei der Bestimmung der Konsequenzen für
```

File: content/version/2/0/code_of_conduct.de.md [hunk 26]
```diff
@@ -0,0 +1,137 @@
...
+jede Handlung, die ihrer Ansicht nach gegen diesen Verhaltenskodex verstößt, an
+diese Richtlinien über die Auswirkungen in der Gemeinschaft halten:
+
+### 1. Berichtigung
+
+**Auswirkungen in der Gemeinschaft**: Verwendung unangemessener Sprache oder
+anderes Verhalten, das in der Gemeinschaft als unprofessionell oder unwillkommen
+gilt.
+
+**Folge**: Eine private, schriftliche Verwarnung von den Gemeinschaftsleitung,
```
Comment (ImmoStanke) [inline content/version/2/0/code_of_conduct.de.md]:
```suggestion
**Folge**: Eine private, schriftliche Verwarnung von der Gemeinschaftsleitung,
```
or, more concisely:
```suggestion
**Folge**: Eine private, schriftliche Verwarnung der Gemeinschaftsleitung,
```

**Review by ImmoStanke — CHANGES_REQUESTED** [review]
Since "Gemeinschaftsleitende" was changed to "Gemeinschaftleitung", the case has to be adjusted everywhere, because "Gemeinschaftsleitung" is singular whereas  "Gemeinschaftsleitende" was plural.
Hopefully I caught all the occurrences.

**Comment by mheob** [comment]
What a terrible mistake on my part 🤦🏽‍♂️. Thanks for the correction @ImmoStanke.

**Comment by transdigiware** [comment]
I want to add a language for bahasa  Indonesian , version by version

**Comment by mheob** [comment]
Do we have more reviews? Or would you say the german version is ok?

**Comment by CoralineAda** [comment]
@mheob Looks like you addressed everything. Merging! Thank you so much (and thanks to all the reviewers!)
